### PR TITLE
Split changelog PR off helm publish workflow

### PR DESCRIPTION
**Pull request by k0da**
Split publish jobs into two independed worflows

Signed-off-by: Dinar Valeev <dinar.valeev@absa.africa>

### CODE REVIEW

File: .github/workflows/changelog_pr.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+name: Open PR with offline changelog
```
Comment (somaritane) [inline .github/workflows/changelog_pr.yaml:1]: Small suggestion:
```suggestion
name: Offline Changelog
```

**Review by somaritane — CHANGES_REQUESTED** [review]
@k0da lgtm, just a small title suggestion

File: .github/workflows/changelog_pr.yaml (hunk 1 above)
Comment (k0da) [reply .github/workflows/changelog_pr.yaml]: done

**Review by somaritane — APPROVED** [review]
lgtm!

**Comment by ytsarev** [comment]
@k0da @somaritane there is still a chance of race condition/deviation for offline CHANGELOG.md. https://docs.github.com/en/actions/guides/storing-workflow-data-as-artifacts might be a way to enhace it

**Pull request merged**